Sources/Plasma/FeatureLib/pfPython/pyEnum.cpp

```diff
@@ -50,212 +50,151 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 #include "HeadSpin.h"
 #include <Python.h>
 #include <structmember.h>
-#include "pyGlueHelpers.h"
 
-#include "pyEnum.h"
 #include <string_theory/format>
 
+#include "pyEnum.h"
+#include "pyGlueHelpers.h"
+#include "pyObjectRef.h"
+
 struct EnumValue {
     PyObject_HEAD
-    long value;
-    char* name;
+    hsSsize_t value{};
```

Comment:
`hsSsize_t` and `Py_ssize_t` are not guaranteed to be the same type...  You should use `Py_ssize_t` (or another type documented/supported by Python) to better guarantee compatibility with the Python API.